via/services/jstor.py

```diff
@@ -3,23 +3,39 @@
 
 
 class JSTORAPI:
+    """An interface for dealing with JSTOR documents."""
+
     DEFAULT_DOI_PREFIX = "10.2307"
-    """Main DOI prefix. Used for real DOIs and non registered pseudo-DOIs with the same format"""
+    """Used when no DOI prefix can be found."""
```

Comment:
I don't understand what this was trying to say before, so I may have phrased this wrong.

Reply:
Yeah this is clearer. As far as I remember DOI without a prefix was not a real DOI but something else. That distinction is completely irrelevant.